src/main/java/york/pharmacy/orders/OrderMapper.java

```diff
@@ -0,0 +1,30 @@
+package york.pharmacy.orders;
+
+import york.pharmacy.orders.dto.OrderRequest;
+import york.pharmacy.orders.dto.OrderResponse;
+
+public class OrderMapper {
+
+    // Convert OrderRequest DTO to Order Entity
+    public static Order toEntity(OrderRequest orderRequest) {
+        Order order = new Order();
```

Comment:
If you use the all args constructor here, any changes to the structure of the order class will be enforced by the compiler and it will catch any potential bugs caused by only a partial refactor:
`new Order(null, orderRequest.getMedId() ,orderRequest.getQuantity(), orderRequest.getDeliveryDate(), OrderStatus.ORDERED, null, null);`